daemon/src/model/cfd.rs

```diff
@@ -721,7 +735,7 @@ impl Cfd {
         // whether they match
         let hours_to_charge = 1;
 
-        let funding_fee = FundingFee::new(self.margin(), self.funding_rate, hours_to_charge)?;
+        let funding_fee = RolloverFee::new(self.margin(), self.funding_rate, hours_to_charge)?;
```

Comment:
I think this would be simpler if we would calculate `FundingFee` here (which is always for 1 hour) and then simply multiply it by `hours_to_charge`.

Reply:
That should also allow you to get rid of the entire `RolloverFee` concept which - as far as I understand - only exists because we might need to charge for more than one hour.

Reply:
> That should also allow you to get rid of the entire `RolloverFee` concept which - as far as I understand - only exists because we might need to charge for more than one hour.

The `RolloverFee` also stores the funding rate, which needs to to be presented in the projection ultimately.

Reply:
I removed one of the "Fee" structs, hopefully it's cleaner now. Now `FundingFee` is the main thing, it can optionally be initialised with `OpeningFee` on top of that, and it can tally up the fees (implements `Add` trait)